sandbox/story.json

```diff
@@ -57,6 +57,22 @@
                 }
             ]
         },
+        {
+            "type": "heading-two",
+            "children": [
+                {
+                    "text": "Soft line-breaks"
```

Comment:
should we also preserve several spaces like 
```
Soft            line-breaks
```

Reply:
> should we also preserve several spaces like

@aspirisen Yes. This is already done in #4